_posts/contests/2021-02-05-Performance-analysis-and-tuning-contest-4.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: Performance analysis and tuning contest &#45&#51.
+categories: contest
+---
+
+**Contents:**
+* TOC
+{:toc}
+
+Welcome to the 4th edition of our performance analysis and tuning contest. If you see this post and haven't read my initial post about the [contest]({{ site.url }}/blog/2019/02/02/Performance-optimization-contest), I encourage you to read it first. 
```

Comment:
Thanks for preparing the post. We need to check all the places where we say "I". Since you are the author, we need to update every other place to "Denis". I will double-check it later.

Reply:
Change to Denis only on those places where it makes sense. I know I copied your article, but it saved a lot of time.